frontend/src/routes.tsx

```diff
@@ -1,60 +1,108 @@
 import { createBrowserRouter } from "react-router-dom";
-import App from './App.tsx'
+import App from './App.tsx';
 import LoginBox from './login/LoginBox.tsx';
 import SignUpBox from './login/SignUpBox.tsx';
 import CenterNavigation from './components/CenterNavigation.tsx';
 import DeckManager from './components/DeckManager.tsx';
-import ListLayout from "./components/ListLayout.tsx";
+import ListLayout from './components/ListLayout.tsx';
+import EnemyUpsert from './components/EnemyUpsert.tsx';
+import MainLayout from './layouts/MainLayout.tsx'; // Import your MainLayout
 
 const router = createBrowserRouter([
   {
     element: <App />,
     children: [
       {
         path: "/",
-        element: <LoginBox />,
+        element: (
+          <MainLayout header="Log in">
```

Comment:
Jeg er ikke sikker på vi behøver at wrappe indholdet af hver element med <MainLayout>. Det ender ihvertfald med at give meget kode dublikation i routeren.

Reply:
Hvis vi gør det i child elementerne kommer vi til at have samme mængde kode, hvorimod her bliver det samlet et sted. Jeg kunne ikke lige komme på en løsning til er fjerne replikeret kode, men er åben over for måde vi kan fjerne det. Kan også være at header bare skal skrottes i main layout, så det kun er den gældende komponent der skal sættes ind. 
Plz giv feedback på hvad i synes